server-mock/db/db.go

```diff
@@ -427,3 +430,44 @@ func (d *Database) MarkValidatorsRegistered(vaultAddress common.Address, network
 
 	return nil
 }
+
+// Call this to set the private key for the ConstellationAdmin contract
+func (d *Database) SetConstellationAdminPrivateKey(privateKey []byte) {
+	d.ConstellationAdminPrivateKey = privateKey
+}
+
+// Call this to set the AvailableConstellationMinipoolCount for a user
+func (d *Database) SetAvailableConstellationMinipoolCount(nodeAddress common.Address, count int) error {
+	for _, user := range d.Users {
+		for _, candidate := range user.RegisteredNodes {
+			if candidate.Address == nodeAddress {
+				user.AvailableConstellationMinipoolCount = count
+				return nil
+			}
+		}
+		for _, candidate := range user.WhitelistedNodes {
```

Comment:
We probably want to leave this out and only apply it to registered nodes, if there isn't one then just error out saying the node address hasn't been registered or something.

Reply:
updated